likeminds_feed_nova_sample/lib/cred_screen.dart

```diff
@@ -118,18 +119,21 @@ class _CredScreenState extends State<CredScreen> {
   @override
   Widget build(BuildContext context) {
     ThemeData theme = Theme.of(context);
+    Size screenSize = MediaQuery.of(context).size;
     // return lmFeed;
     userId = null; // UserLocalPreference.instance.fetchUserId();
     // If the local prefs have user id stored
     // Login using that user Id
     // otherwise show the cred screen for login
     if (userId != null && userId!.isNotEmpty) {
       return lmFeed = LMFeed.instance(
-        userId: userId,
-        userName: 'Test User',
-        callback: LikeMindsCallback(),
-        apiKey: "",
-      );
+          userId: userId,
+          userName: 'Test User',
+          imageUrl:
```

Comment:
Don't use imageUrl in our sample app, let the customer use it accordingly

Reply:
done